tests/testProcessCcd.py

```diff
@@ -183,9 +184,12 @@ def assertCatalogsEqual(self, catalog1, catalog2, skipCols=()):
         self.assertEqual(catalog1.schema, catalog2.schema)
         self.assertEqual(len(catalog1), len(catalog2))
         d = catalog1.schema.extract("*")
+        def fixNaN(x):
+            if x!=x:
+                return "NaN"
         fixNaN = lambda x: x if x == x else "NaN"
```

Comment:
Yes, I think I was in the middle of making this change when something interrupted me and since it didn't break anything I must have forgotten about it. Great catch.